flag.go

```diff
@@ -19,7 +19,7 @@ type Flag struct {
 	DefaultValue interface{}
 }
 
-// Return the name of the flag preceding the right amount of dashes.
+// Name Return the name of the flag preceding the right amount of dashes.
```

Comment:
You probably ran `go vet` or something here. Please adjust the comment to a proper English sentence (“Name *r*eturn*s*...”)